Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -249,7 +249,7 @@ namespace pcpp
 			PcapStats stats;
 			getStatistics(stats);
 			m_cbOnStatsUpdate(stats, m_cbOnStatsUpdateUserCookie);
-			multiPlatformSleep(m_IntervalToUpdateStats);
+			std::this_thread::sleep_for(std::chrono::milliseconds(m_IntervalToUpdateStats));
```

Comment:
This should be:
```cpp
std::this_thread::sleep_for(std::chrono::seconds(m_IntervalToUpdateStats));
```

Reply:
Fixed with a4c4d6dcec0f643c665227ed24d21d944b865aa7